utils/sendEmail.js

```diff
@@ -0,0 +1,74 @@
+require('dotenv').config();
+const AWS = require('aws-sdk');
+
+const SESConfig = {
+  apiVersion: '2010-12-01',
+  accessKeyId: process.env.AWS_SES_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SES_SECRET_ACCESS_KEY,
+  region: process.env.AWS_SES_REGION
+};
+
+const params = {
+  Source: process.env.AWS_SES_SOURCE_EMAIL,
+  Destination: {
+    ToAddresses: [process.env.AWS_SES_DESTINATION_EMAIL]
+  },
+  ReplyToAddresses: [process.env.AWS_SES_SOURCE_EMAIL],
+  Message: {
+    Body: {
+      Html: {
+        Charset: 'UTF-8',
+        Data: ''
+      }
+    },
+    Subject: {
+      Charset: 'UTF-8',
+      Data: `Absence Report - ${new Date()}`
```

Comment:
We might need to change the date format depending on the date returned when the function runs on aws.